### DISCUSSION TITLE
icons in main screen

### DISCUSSION THREAD

**Discussion by mariush444** [CatimaLoyalty/Android#756] [Ideas]
Is it possible to add to display icons (picture, taken photo) on the main screen instead off or additionally to text?

**Comment by TheLastProject** [comment]
This has been possible since Catima [2.9.0](https://github.com/CatimaLoyalty/Android/releases/tag/v2.9.0).

Just tap the letter in the edit screen and choose either "Take a photo" or "Select image from gallery":
![image](https://user-images.githubusercontent.com/1885159/150124695-89c3cbd4-56c1-4748-9391-71e5f9223229.png)
